docs/src/pages/guides/batfish-helpers.md

```diff
@@ -252,6 +253,69 @@ class PageShell extends React.Component {
 {{JSON.stringify(splitPages, null,2)}}
 ```
 
+## Prepare sitemap
+
+`prepareSitemap` creates a data file that lists all files with `hideFromSearchEngines: true` or `splitPage: true` set in the page's frontMatter. Use this generated file with Batfish's [`sitemap.ignoreFile`](https://github.com/mapbox/batfish/blob/main/docs/configuration.md#sitemap) option to generate a sitemap that excludes any paths in the prepared file.
+
+### Arguments
+
+- `data`, object. Provided by the data selector.
+- `siteBasePath`, string. The site's `siteBasePath` as defined in the Batfish config.
+- `outputDirectory`, string. Batfish's `outputDirectory`. Default `_site`.
+- `docsPath`, string. The directory that `batfish.config.js` lives in, if not in the root. (It's unlikely that you will use this option.)
+
+### Set up in batfish.config.js
+
+```js
+const { prepareSitemap } = require('@mapbox/dr-ui/helpers/batfish/index.js');
+
+module.exports = () => {
+  return {
+    sitemap: {
+      // generated by the sitemapIgnore dataSelector
+      ignoreFile: '_site_tmp/data/sitemap-ignore.js'
```

Comment:
Similar question here: you can use `sitemap: true` if you don't want to exclude some pages from the sitemap, right?

Reply:
If you don't want to intervene with the sitemap, you don't need to specific `sitemap` at all. For Batfish, the sitemap will automatically build if you define `siteOrigin`, ref: https://github.com/mapbox/batfish/blob/main/docs/configuration.md#sitemap